src/app/auth/views/Auth/AuthView.tsx

```diff
@@ -29,101 +30,104 @@ const SignupAuth = () => {
     },
   });
   return (
-    <div className="flex w-full max-w-lg flex-col items-center space-y-2 pt-10 lg:w-max lg:items-start lg:pt-0">
-      <div className="flex w-full flex-col space-y-3 lg:flex-row lg:space-y-0 lg:space-x-3">
-        <div className="flex w-full">
-          <TextInput
-            placeholder={'Correo'}
-            label="email"
-            type="email"
-            className={'w-full'}
-            register={register}
-            minLength={{ value: 1, message: 'El correo no puede estar vacío' }}
-            error={errors.email}
-          />
-        </div>
+    <form
+      onSubmit={handleSubmit(() =>
+        signup(
+          {
+            email: localStorage.getItem('email'),
+            password: localStorage.getItem('password'),
+          },
+          dispatch,
+          doRegister,
+          setLoading,
+          setError,
+        ),
+      )}
+    >
+      <div className="flex w-full max-w-lg flex-col items-center space-y-2 pt-10 lg:w-max lg:items-start lg:pt-0">
+        <div className="flex w-full flex-col space-y-3 lg:flex-row lg:space-y-0 lg:space-x-3">
+          <div className="flex w-full">
+            <TextInput
+              placeholder={'Correo'}
+              label="email"
+              type="email"
+              className={'w-full'}
+              register={register}
+              minLength={{ value: 1, message: 'El correo no puede estar vacío' }}
```

Comment:
I assume there is no need to add translations because it is an element of a campaign with PCComponentes and it is only in Spanish.